lib/gradle.properties

```diff
@@ -1,2 +1,8 @@
 kotlin.code.style=official
+signing.keyId=6586A7EC
+signing.password=larkin
```

Comment:
Don't check these thing in. Instead, we should disable signing/publishing for dev work flows and only turn this on when doing a publish.